__tests__/articles.test.js

```diff
@@ -79,4 +79,73 @@ describe('GET /api/articles', () => {
             });
         });
     });
-});
\ No newline at end of file
+});
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('200: responds with the updated article', () => {
+        const incVotes = { inc_votes: 1 };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(incVotes)
+            .expect(200)
+            .then((response) => {
+                const { article } = response.body;
+                expect(article).toEqual(
+                    expect.objectContaining({
+                        article_id: 1,
+                        votes: expect.any(Number)
+                    })
+                )
+                expect(article.votes).toBe(101);
+            })
+    })
+
+    test('200: responds with the updated article when votes are decremented', () => {
+        const incVotes = { inc_votes: -100 };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(incVotes)
+        .expect(200)
+        .then((response) => {
+            const { article } = response.body;
+            expect(article).toEqual(
+                expect.objectContaining({
+                    article_id: 1,
+                    votes: expect.any(Number)
+                })
+            );
+            expect(article.votes).toBe(0);
+        });
+    });
+
+
+    test('400: responds with an error message when inc_votes is missing', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({})
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad request');
+        });
+    });
+
```

Comment:
We could have one more test in here. What error would we expect if inc_votes is not a number?